File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/FreightTemplateService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbFreightTemplate;							  
import entity.PageResult;
import entity.Result;
/**
 * freightTemplate服务层接口
 * @author jieweili
 *
 */
public interface FreightTemplateService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbFreightTemplate> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param freightTemplate
     * @return
     */
    public Result saveFreightTemplate(TbFreightTemplate freightTemplate);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbFreightTemplate getFreightTemplateById(Long id);

    /**
     * 修改对象
     * @param freightTemplate
     * @return
     */
    Result updateFreightTemplate(TbFreightTemplate freightTemplate);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeFreightTemplate(Long[] idList);

    /**
     * 根据条件分页查询
     * @param freightTemplate
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listFreightTemplateByExample(TbFreightTemplate freightTemplate, Integer pageNum, Integer pageSize);
 
}

File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/freightTemplateService.js
app.service('freightTemplateService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../freightTemplate/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../freightTemplate/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeFreightTemplateList = function (idList) {
        return $http.get('../freightTemplate/removeFreightTemplate.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../freightTemplate/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateFreightTemplate = function (freightTemplate) {
        return $http.post('../freightTemplate/updateFreightTemplate.do', freightTemplate)
    };

    //新增品牌
    this.saveFreightTemplate = function (freightTemplate) {
        return $http.post('../freightTemplate/saveFreightTemplate.do', freightTemplate)
    };   
});
